test/braket_tests/base/test_jobs_qaoa.py

```diff
@@ -12,10 +12,13 @@
 # language governing permissions and limitations under the License.
 
 import time
+import pytest
+
 from ..common.braket_jobs_util import job_test
 
 
-def test_qaoa_circuit(account, role, s3_bucket, image_list):
+@pytest.mark.parametrize("decorator", (False, True))
```

Comment:
I think it's overkill to test with and without decorators when we're just trying to see if the containers are valid.